master/security/app-layer-policy/index.md

```diff
@@ -3,22 +3,23 @@ title: Application layer policy tutorial
 canonical_url: 'https://docs.projectcalico.org/v3.7/security/app-layer-policy/index'
 ---
 
-The included demo sets up a microservices application, then demonstrates the use of application
-layer policy to mitigate some common threats.
+This tutorial sets up a microservices application, then demonstrates how to use {{site.prodname}} application layer policy to mitigate some common threats.
 
 ## Prerequisites
 
-To create a Kubernetes cluster which supports application layer policy, follow
-one of our [getting started guides]({{site.baseurl}}/{{page.version}}/getting-started/),
-and [enable application layer policy]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/app-layer-policy).
-
-You will also need the [calicoctl]({{site.baseurl}}/{{page.version}}/getting-started/calicoctl/install) command line tool.
+1. Build a Kubernetes cluster.
+1. Install Calico on Kubernetes:  
+- If Calico is not installed on Kubernetes, see [Calico on Kubernetes]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes).  
+- If Calico is already installed on Kubernetes, verify that you have: Calico networking with [Kubnernetes Data Store over etcd (recommended)]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/calico) or a non-Calico CNI, and Calico network policy. 
```

Comment:
I don't think the "recommended" bit here is right - I don't think we would say etcd is recommended over other installation approaches.

Reply:
Also, I don't think this tutorial requires that etcd be in use

Reply:
Spike confirmed that you are correct. Either works. Fixed text.